WooCommerce/UITestsFoundation/Screens/Products/ProductsScreen.swift

```diff
@@ -17,13 +19,12 @@ public class ProductsScreen: ScreenObject {
         (try? ProductsScreen().isLoaded) ?? false
     }
 
-    init(app: XCUIApplication = XCUIApplication()) throws {
+    public init(app: XCUIApplication = XCUIApplication()) throws {
```

Comment:
Now, once the asserting method is inside the class, there is no longer a need for this change (same as for `SingleProductScreen.swift`).